engine/api/src/main/java/pl/touk/nussknacker/engine/api/editor/SimpleEditor.java

```diff
@@ -0,0 +1,15 @@
+package pl.touk.nussknacker.engine.api.editor;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.PARAMETER)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface SimpleEditor {
+
+    SimpleEditorType type();
+
+    LabeledExpression[] possibleValues() default {};
```

Comment:
Can you put here a note that this field is for `FIXED_VALUES_EDITOR`?

Reply:
Done